README.md

```diff
@@ -2,57 +2,45 @@
 
 > **Test CLI Applications** - This crate checks the output of a child process is as expected.
 
-[![Build Status](https://travis-ci.org/killercup/assert_cli.svg)](https://travis-ci.org/killercup/assert_cli) [![Documentation](https://img.shields.io/badge/docs-master-blue.svg)][Documentation]
+[![Build Status](https://travis-ci.org/killercup/assert_cli.svg)](https://travis-ci.org/killercup/assert_cli) [![Documentation](https://img.shields.io/badge/docs-master-blue.svg)][Documentation] ![License](https://img.shields.io/crates/l/assert_cli.svg) [![crates.io](https://img.shields.io/crates/v/assert_cli.svg)](https://crates.io/crates/assert_cli)
 
 ## Install
 
-Just add it to your `Cargo.toml`:
+For your tests, just add it to your `Cargo.toml`:
 
 ```toml
-[dependencies]
+[dev-dependencies]
 assert_cli = "0.5"
 ```
 
 ## Example
 
 Here's a trivial example:
 
-```rust
+```rust,ignore
```

Comment:
Huh? Does this not work in skeptic? We can tweak the skeptic test setup if you want.

Reply:
Yes, it will report in `cargo test`'s output as a skipped test which is awesome because you know there is a code example to be cautious of.

Reply:
Yes, that's what ignored tests do. I was asking why it's ignored and how we can make it not-ignored :)

Reply:
Oh, sorry.

Its buried in the commits if you find the right one :)

>  Sadly, `main_binary` needs to be skipped because skeptic has the cargo stuff locked.

Unfortunately, to discover why it failed, I had to turn off `--quiet`.

Reply:
Oh, okay. Too bad. Is this a known issue with skeptic?

Reply:
Weird, when I tried to do this before, I thought the error was different than this

> error: could not find `Cargo.toml` in `/tmp/rust-skeptic.eAWTVVXUenGE` or any parent directory

This is related to skeptic using a different working directory which also deserves an issue.

Reply:
Created an issue https://github.com/budziq/rust-skeptic/issues/69

Reply:
I'm very confused over what skeptic does that is not just code block copypasta in a build script and why that fails o_O